cfgov/v1/atomic_elements/molecules.py

```diff
@@ -317,27 +317,32 @@ class Meta:
         label = 'Phone'
 
 
-class ImageInset(blocks.StructBlock):
+class ContentImage(blocks.StructBlock):
     image = atoms.ImageBasic()
-    image_position = blocks.ChoiceBlock(choices=[('right', 'right'),
-                                                 ('left', 'left')],
-                                        default='right')
-    is_image_decorative = blocks.BooleanBlock(required=False,
-                                              label='Image decorative')
-    image_width = blocks.ChoiceBlock(choices=[(170, '170px'),
-                                              (270, '270px')],
-                                     default=270,
-                                     label='Image Width',
-                                     help_text='Default is 270px.')
-    text = blocks.RichTextBlock(required=False)
-    is_bottom_rule = blocks.BooleanBlock(required=False,
-                                         default=True,
-                                         label='Bottom Rule')
+    image_position = blocks.ChoiceBlock(
+        choices=[('right', 'right'),
+                 ('left', 'left')],
+        default='right',
+        help_text='Does not apply if the image is full-width'
+    )
+    image_width = blocks.ChoiceBlock(
+        choices=[('full', 'full'),
+                 (470, '470px'),
+                 (270, '270px'),
+                 (170, '170px')],
+        default='full',)
+    text = blocks.RichTextBlock(required=False, label='Caption')
+    is_bottom_rule = blocks.BooleanBlock(
+        required=False,
+        default=True,
```

Comment:
If it's not required should it have a default?

Reply:
If we want it to be on by default, yes. It's a checkbox.

Reply:
Ohhh, was thinking Boolean was a radio button. Carry on.